pkg/apis/camel/v1/integrationkit_types.go

```diff
@@ -57,6 +57,7 @@ type IntegrationKitStatus struct {
 // +kubebuilder:printcolumn:name="Phase",type=string,JSONPath=`.status.phase`,description="The integration kit phase"
 // +kubebuilder:printcolumn:name="Type",type=string,JSONPath=`.metadata.labels.camel\.apache\.org\/kit\.type`,description="The integration kit type"
 // +kubebuilder:printcolumn:name="Image",type=string,JSONPath=`.status.image`,description="The integration kit image"
+// +kubebuilder:resource:path=integrationkits,scope=Namespaced,shortName=ik,categories=kamel;camel
```

Comment:
Sorry I may not have be enough specific. The `kubebuilder:resource` marker already exists few lines above (55). Only the `categories` should be appended to the existing marker, instead of duplicating it.